### DISCUSSION TITLE
My system can run whisper with python3 but 'command not found' on the terminal [Mac]

### DISCUSSION THREAD

**Discussion by Perthz** [openai/whisper#162] [General]
I spent a few hours trying to use the command line but gave up since Python3 is also convenient. 

However, does everyone familiar with this problem?

**Comment by glangford** [comment]
On OS X 12.6, if I do
```
which whisper
```
from the shell, it says
```
/Library/Frameworks/Python.framework/Versions/3.10/bin/whisper
```
For whatever version of Python you have, it might be that the /bin directory of Python is not in your $PATH. Can you check that?

**Reply by Perthz** [reply]
Thanks for the response.

I ran

`which whisper
`

Result 

`whisper not found
`

I've tried to follow your path /Library/Frameworks/Python.framework/Versions/3.10/bin/whisper, 

mine is 3.9, but no whisper in /bin 

<img width="595" alt="Screenshot on 2022-09-29 at 02-49-09" src="https://user-images.githubusercontent.com/3999645/192798692-96fad49e-38b7-4cab-909d-e1f53181a46d.png">

I can find whisper package here. /Library/Python/3.9/lib/python/site-packages/whisper/

**Reply by glangford** [reply]
So now if you
```
echo $PATH
```
it is likely that none of the directories contain whisper. You can try copying whisper to one of the directories in your PATH, or updating PATH.

**Reply by Perthz** [reply]
Thank you! You guided me to the right direction. I tried to use 'pip3 install --prefix' to the right path.

It did not work the first time, uninstall and reinstall python. After a few tries, it's now on the correct path.
